### DISCUSSION TITLE
Hello there!

### DISCUSSION THREAD

**Discussion by Abigblueworld** [community/community#129136] [New to GitHub]
### Select Topic Area

General

### Body

<b> My name is Blue, or BlueWorld! </b>

I am a 17 (on June 21) year old high-fuctioning autistic female HTML & CSS programmer!

_Say_, is anyone else on here autistic and good at the _website programming_? I would be happy to say hi! 

If so, do you guys have a **_<u> super kuul </u>_** website? 

_<s> I hope you might not ask me </s>_

(Sorry for my slang, I'm literally a teenager with an age regressing brain)

**Comment by ebndev** [comment]
Welcome to GitHub and the GitHub Community @Abigblueworld 🎆! We're glad you're here and can't wait to see where your GitHub journey takes you.

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋

**Reply by Abigblueworld** [reply]
My bad, sorry.

I just like introducing myself like this.
